src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/matmul_quantized_subgraph.cpp

```diff
@@ -152,12 +187,33 @@ INSTANTIATE_TEST_SUITE_P(smoke_matmulBrgemmInt8,
                          MatmulBrgemmInt8Test,
                          ::testing::Combine(::testing::ValuesIn(supportedInputShapes),
                                             ::testing::ValuesIn({true, false}),
+                                            ::testing::Values(false),
                                             ::testing::ValuesIn({ElementType::u8, ElementType::i8}),
                                             ::testing::ValuesIn({ElementType::f32, ElementType::u8, ElementType::i8}),
-                                            ::testing::ValuesIn(matmulSpecificFilterParams)),
+                                            ::testing::ValuesIn(matmulSpecificFilterParams),
+                                            ::testing::Values(ov::AnyMap{})),
                          MatmulBrgemmInt8Test::getTestCaseName);
 
-}  // namespace
+std::vector<ov::AnyMap> additionalConfig = {{ov::hint::inference_precision(ov::element::f32)},
+                                            {ov::hint::inference_precision(ov::element::bf16)},
+                                            {ov::hint::inference_precision(ov::element::f16)}};
 
+const std::vector<CPUSpecificParams>fullyconnectedSpecificFilterParams = {
+    {{}, {}, {"brgemm_avx512_amx"}, "brgemm_avx512_amx"},
+    {{}, {}, {"jit_gemm"}, "jit_gemm"}
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_fullyconnected_prim_impl_type_check,
+                         MatmulBrgemmInt8Test,
+                         ::testing::Combine(::testing::Values(ov::Shape{1, 32, 36}),
+                                            ::testing::Values(true),
+                                            ::testing::Values(true),
+                                            ::testing::Values(ElementType::u8),
+                                            ::testing::Values(ElementType::u8),
+                                            ::testing::ValuesIn(fullyconnectedSpecificFilterParams),
+                                            ::testing::ValuesIn(additionalConfig)),
+                         MatmulBrgemmInt8Test::getTestCaseName);
+
+}  // namespace
```

Comment:
I didn't see the test case of checking to extra reorder or output precision to be U8/S8 not FP32. My understanding just adding the bias and check the implement type. Did I miss anything?
Also I doubted whether need to add test case to check the output precision mapping behavior. Usually just need to ensure the table is aligned with onednn spec. Need @dmitry-gorokhov to decide whether the test case is needed.
If test case needed, maybe just need to add some new test case of checking the runtime graph/execution graph in my understanding. Some code like below: 

https://github.com/openvinotoolkit/openvino/blob/c4d6d2b37db873e7c657a6049f858cf1cf472cb8/src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/matmul_weights_decompression.cpp#L180

Reply:
Seems to be minor change -> would propose to avoid any test changes in this PR

Reply:
have removed the testcase changes in this pr